src/screens/Profile/Home/index.tsx

```diff
@@ -153,7 +153,31 @@ export default function ProfileHome({navigation}: ProfileScreenProps) {
             {authenticationType === 'phone' && (
               <Sections.LinkItem
                 text={t(ProfileTexts.sections.account.linkItems.logout.label)}
-                onPress={signOut}
+                onPress={() =>
+                  Alert.alert(
+                    t(ProfileTexts.sections.account.linkItems.logout.confirm),
+                    undefined,
+                    [
+                      {
+                        text: t(
+                          ProfileTexts.sections.account.linkItems.logout.alert
+                            .cancel,
+                        ),
+                        style: 'cancel',
+                      },
+                      {
+                        text: t(
+                          ProfileTexts.sections.account.linkItems.logout.alert
+                            .confirm,
+                        ),
+                        style: 'destructive',
+                        onPress: () => {
+                          signOut();
+                        },
+                      },
+                    ],
+                  )
+                }
```

Comment:
Kan det være greit å dra dette ut i en egen funksjon, så tar den ikke så mye plass her?

Reply:
God idé! Skal se hva som kan trekkes ut 👌

Reply:
Gjorde et forsøk nå. To spørsmål:

1. Tenker du funksjonen burde flyttes enda lenger ut? Har du i så fall et forslag til hvor?
2. Burde vi være mer restriktive enn `any` på hva som er tillatt å være en `destructiveArrowFunction`?

Reply:
1. Var noe slik som dette jeg tenkte 👍 Alternativt kunne vi lagd en `utils.ts`-fil i Home-mappen som eksporterer en destructiveAlert, men dette funker, så opp til deg.
2. `() => void` er vel en mer riktig type her?

Og så får vi lint-feil ser jeg, så kanskje like greit å wrappe tekstene i `t()` før de sendes inn i funksjonen, og heller sette typen til `string`.

Reply:
Dytta den ut til `utils.ts` jeg, kanskje vi får behov for den et annet sted en gang.

`() => void` fungerte som en kule, litt mer spesifikt i hvert fall 👌